File: app/src/main/java/org/jaaksi/pickerview/demo/MainActivity.java
package org.jaaksi.pickerview.demo;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.View;

/**
 * Created by fuchaoyang on 2018/2/10.<br/>
 * description：
 */

public class MainActivity extends BaseActivity {

  @Override
  protected void onCreate(@Nullable Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_main);
  }

  public void onClick(View view) {
    switch (view.getId()) {
      case R.id.btn_timepicker:
        openFragment(new TimePickerFragment());
        break;
      case R.id.btn_mixedtimepicker:
        openFragment(new MixedTimeFormatFragment());
        break;
      case R.id.btn_optionpicker:
        openFragment(new OptionPickerFragment());
        break;
      case R.id.btn_test_pickerview:
        openFragment(new TestPickerViewFragment());
        break;
      case R.id.btn_picker_configs:
        openFragment(new PickerConfigFragment());
        break;
    }
  }
}

File: app/src/main/java/org/jaaksi/pickerview/demo/ui/MainActivity.kt
package org.jaaksi.pickerview.demo.ui

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.Fragment
import org.jaaksi.pickerview.demo.R

/**
 * Created by fuchaoyang on 2018/2/10.<br></br>
 * description：
 */
class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun onClick(view: View) {
        when (view.id) {
            R.id.btn_timepicker -> openFragment(TimePickerFragment())
            R.id.btn_mixedtimepicker -> openFragment(MixedTimeFormatFragment())
            R.id.btn_optionpicker -> openFragment(OptionPickerFragment())
            R.id.btn_optionpicker2 -> openFragment(OptionPickerFragment2())
            R.id.btn_test_pickerview -> openFragment(TestPickerViewFragment())
        }
    }

    private fun openFragment(fragment: Fragment) {
        supportFragmentManager.beginTransaction()
            .add(android.R.id.content, fragment)
            .addToBackStack(null)
            .commit()
    }
}